colrev/env/language_service.py

```diff
@@ -35,8 +35,9 @@ def compute_language(self, *, text: str) -> str:
         """Compute the most likely language code"""
         language = self.__lingua_language_detector.detect_language_of(text)
         if language:
-            return language.name.lower()
-        return language
+            return language.iso_code_639_3.name.lower()
```

Comment:
We could probably also use these built_in iso codes for the `__lang_code_mapping`. Would save us the pycountry dependency. Would be based on languages rather than countries though. Not sure whether that is a problem. Maybe something like this:

```python
self.__lang_code_mapping = {}
for lang in lingua.Language.all():
    self.__lang_code_mapping[lang.name.lower()] = lang.iso_code_639_3.name.lower()
```